packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx

```diff
@@ -83,8 +87,8 @@ export class Dialog extends BaseComponent<IDialogProps, IDialogState> {
         onDismissed={ onDismissed }
         onLayerDidMount={ onLayerDidMount }
         responsiveMode={ responsiveMode }
-        subtitleAriaId={ subText && id + '-subText' }
-        titleAriaId={ title && id + '-title' }
+        subtitleAriaId={ this._getAriaLabelId(ariaDescribedById, subText, id + 'subText') }
```

Comment:
I think you have to bring back - before `subText` to keep backward compatible, don't you?